smartmin/views.py

```diff
@@ -420,6 +420,8 @@ class SmartTemplateView(SmartView, TemplateView):
 
 
 class SmartReadView(SmartView, DetailView):
+    slug_field = None
+    slug_url_kwarg = None
```

Comment:
I started by adding a single `object_by_uuid` field but then thought it would be good at the smartmin level to allow slug fields other than UUID. Maybe `slug_url_kwarg` can default to the value of `slug_field` and so views only have to specify the latter.. hmm